src/main/java/frc/robot/commands/basic/teleopCmd.java

```diff
@@ -11,33 +11,40 @@ public class teleopCmd extends CommandBase {
   @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
   private final drivetrainSubsystem drivesubsystem;
 
-  private final SlewRateLimiter slrForTurn =
-      new SlewRateLimiter(Constants.SLEWTURN); // from 2 to 2.5 to 3.5 to 4.0 to 4.5
+  private double turns;
+  private double speeds;
+  private final Joystick driver1 = new Joystick(Constants.DRIVER_ONE_PORT);
+  private final Joystick driver2 = new Joystick(Constants.DRIVER_SEC_PORT);
+  private final SlewRateLimiter slrForTurn = new SlewRateLimiter(Constants.SLEWTURN);
   private final SlewRateLimiter slrForDrive = new SlewRateLimiter(Constants.SLEWSPEED);
 
-  private final Joystick stick = new Joystick(Constants.DRIVER_PORT);
   private boolean isTeleOp = false;
```

Comment:
`isTeleOp` variable is unused.

Reply:
it isn't, down at the bottom of the file it uses it to determine when to stop the program